Apps/ValidationTests/Win32/App.cpp

```diff
@@ -270,7 +270,7 @@ LRESULT CALLBACK WndProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam)
             }
             break;
         }
-        case WM_DESTROY:
+        case WM_CLOSE:
```

Comment:
Not sure about this one. I'm assuming you are seeing some shutdown issues. Usually the sequence is WM_CLOSE calls DestroyWindow which then causes WM_DESTROY which calls PostQuitMessage.

Reply:
Replacing with direct errorCode for postquitmessage instead of using message param.